Cargo.toml

```diff
@@ -15,23 +15,21 @@ readme = "README.md"
 
 iron = "^0.5.0"
 #handlebars = { path = "../handlebars", default-features = false }
-handlebars = { version = "^0.25.0", default-features = false }
-rustc-serialize = { version = "^0.3.16", optional = true }
-serde = { version = "^0.9.0", optional = true }
-serde_json = { version = "^0.9.0", optional = true }
+handlebars = { version = "^0.26.1", default-features = false }
```

Comment:
No need to turn off default-features any more. Just use `handlebars = "^0.26.1" here.